client/ayon_hiero/api/launchforhiero.py

```diff
@@ -24,6 +24,10 @@ def _hiero_main_menubar():
         QtWidgets.QMenuBar
     )]
 
+    # OSX fix - hiero_window.children() can't find menubars
+    if not menubar:
+        menubar = [hiero_window.menuBar()]
+
     assert len(menubar) == 1, "Error, could not find menu bar!"
     return menubar[0]
```

Comment:
Does `return hiero_window.menuBar()` not work also on Windows and Linux? Because if it does work - let's unify the behavior.


```suggestion
    return hiero_window.menuBar()]
```

Reply:
I tested on Linux and it works.
I don't know if I'm able to unify the code, so I simply added the fix instead 😅
I'm totally ok with you!